cumulus/polkadot-parachain/src/main.rs

```diff
@@ -14,11 +14,66 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Cumulus test parachain collator
+//! Polkadot parachain-omni-node.
 
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
```

Comment:
Re-listing my list of TODOs here, which could be reasonable items to include here, or if not, tracked in issues for the future. We should brainstrom all of the issues related to omni-node, put them all in issues, and create a project board for it to also give visibility about what we will work on next. 

```
// TODO: help in the relay chian args should also imply this
// TODO: ensure execution is not set to native anywhere
// TODO: ensure aura-api is available.
```

Also, adding one related to the recent discovery of blocknumber: 

```
// TODO: metadata check: create the scaffold needed to fetch the metadata, and assert on
// certain types. For example, block number being u32
```

Reply:
> // TODO: ensure execution is not set to native anywhere

This is not the case anymore, since quite some time. No need to have a TODO for this.

Reply:
The CLI flag is still there, is there a reason for that? The docs also still offer native as an option.

Reply:
> The CLI flag is still there, is there a reason for that? The docs also still offer native as an option.

Backwards compatibility. It doesn't do anything anymore and just prints an error.